core-services/prow/02_config/openshift/aws-pod-identity-webhook/_prowconfig.yaml

```diff
@@ -51,6 +51,65 @@ tide:
     - needs-rebase
     repos:
     - openshift/aws-pod-identity-webhook
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - docs-approved
+    - lgtm
+    - px-approved
+    - qe-approved
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - openshift/aws-pod-identity-webhook
+  - excludedBranches:
```

Comment:
you don't need to add this excluded branches section.

for any branch that's not "main", "master", "release-*" or "openshift-*", lgtm+approve is fine (no one should be using branches like that for anything that matters)